setup.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
As discussed, we won't be using solr for now. You can keep your work continue on SQlite only :) 
@Wolf-Legend

Reply:
Ok will remove solr references